BurnOutSharp/PackerType/PECompact.cs

```diff
@@ -30,9 +30,13 @@ public string CheckContents(string file, byte[] fileContent, bool includePositio
             return MatchUtil.GetFirstMatch(file, fileContent, matchers, includePosition);
         }
 
+        // TODO: Improve version detection, Protection ID is able to detect ranges of versions. For example, 1.66-1.84 or 2.20-3.02.
         public static string GetVersion(string file, byte[] fileContent, List<int> positions)
         {
-            return $"v{BitConverter.ToInt16(fileContent, positions[0] + 4)}";
+            short version = BitConverter.ToInt16(fileContent, positions[0] + 4);
+            if (version == 0)
+                return "PE Compact 2";
+            return $"Internal Version {version}";
```

Comment:
Why did you change it to `Internal Version`? Would it make more sense to have it be `(Internal Version {version})` instead?

Reply:
From a conversation in Discord:
```
as for the internal version thing
that's what PiD does
i think its like a beta version thing or something
so a typical PE Compact version is like 1.XX or something
```

Reply:
I still would still like to see this in parenthesis, especially if this is coming after `PE Compact 2`. Personal preference, of course.